packages/kmod-6.1-nvidia-r570/kmod-6.1-nvidia-r570.spec

```diff
@@ -275,6 +330,26 @@ install kernel-open/nvidia-peermem.ko %{buildroot}%{_cross_datadir}/nvidia/open-
 install kernel-open/nvidia-drm.ko %{buildroot}%{_cross_datadir}/nvidia/open-gpu/drivers/
 # end open driver
 
+%if "%{_cross_arch}" == "x86_64"
+# GRID driver
```

Comment:
nit: you could remove these for consistency with the other similar section where the comments were deleted

Reply:
I like these comments here because it helps the reader see the difference in the logic for the 3 builds for each module. Since the logic is different depending on if it's using `kernel` or `kernel-open`, I like keeping them here. In the `%files` section I agree the comments don't add anything, but I do find them useful in the install section.